kano_apps/MainWindow.py

```diff
@@ -94,6 +89,26 @@ def refresh(self, category=None):
     def _app_loaded(self, widget):
         if self._install is not None:
             self._install_apps()
+        elif self._tutorial:
+            self._show_icon_tutorial()
+
+    def _show_icon_tutorial(self):
+        kdialog = KanoDialog(
+            "Add more apps to the desktop",
```

Comment:
@mkeegan when clicking in the [+] symbol in the desktop. This is a "mini tutorial" on how to add icons to the desktop